c/src/ml-api-common.c

```diff
@@ -1169,6 +1169,28 @@ _ml_replace_string (gchar * source, const gchar * what, const gchar * to,
   return result;
 }
 
+/**
+ * @brief Converts predefined entity.
+ */
+gchar *
+_ml_convert_predefined_entity (const gchar * str)
+{
+  gchar *converted = g_strdup (str);
+
+#if defined(__ANDROID__)
+  {
+    extern const char *nnstreamer_native_get_data_path (void);
+
+    const char *data_path = nnstreamer_native_get_data_path ();
```

Comment:
Is ```const char *data_path``` nullable ?

Reply:
yes, it would be null when failed to initialize nnstreamer on android. If data path is null, _ml_replace_string() returns original string.

Reply:
Thank you.